utils/messages.ts

```diff
@@ -47,9 +47,18 @@ Se vuoi, puoi fare subito una simulazione o la richiesta online sul sito di INPS
   EligibilityCheckConflict: () =>
     ({
       subject: "Abbiamo completato le verifiche sul tuo ISEE",
-      markdown: `INPS ha completato le verifiche sull’ISEE e risulta che qualcuno del tuo nucleo familiare abbia già richiesto il Bonus Vacanze.  
+      markdown: `INPS ha completato le verifiche sull’ISEE e risulta che qualcuno del tuo nucleo familiare ha già richiesto il Bonus Vacanze.  
 
 Il Bonus è in fase di attivazione. Ti manderemo un messaggio quando sarà attivo.
+`
+    } as MessageContent),
```

Comment:
Consider remove this cast using something like

```typescript
    MessageContent.decode({
      subject: "Abbiamo completato le verifiche sul tuo ISEE",
      markdown: `INPS ha completato le verifiche sull’ISEE e risulta che qualcuno del tuo nucleo familiare ha già richiesto il Bonus Vacanze.  

Il Bonus è in fase di attivazione. Ti manderemo un messaggio quando sarà attivo.
`
    }).getOrElseL(e => {
      throw new Error(readableReport(e));
    })
```

Reply:
Maybe we'll do this refactor into an other PR ;)